source/content-authors/content-authors-navigating-studio.rst

```diff
@@ -299,6 +299,20 @@ This pencil allows you to edit the content of the wired region, including the cu
     :align: center
     :alt: Navigating Studio - Preview In-Context Editing
 
+To turn off in-context editing, click on the ``Switch off editing`` off switch icon at the top, which will then turn the ``Edit mode``/``Move mode`` icon to gray.
+
+Convenient keyboard shortcuts are provided for turning on/off in-context editing:
```

Comment:
Convenient keyboard shortcuts are also provided, press `?` to see the list of commands.